src/raftstore/coprocessor/dispatcher.rs

```diff
@@ -251,14 +232,15 @@ impl CoprocessorHost {
         );
     }
 
-    pub fn new_split_checker_host(
+    pub fn new_split_checker_host<'a>(
         &self,
+        cfg: &'a Config,
```

Comment:
seems this explicit lifetime is not necessary

Reply:
Because this function accept four reference, lifetime elision do not work here, this explicit lifetime is necessary.